service/src/test/java/bio/terra/cbas/controllers/VerifyPactsAllControllers.java

```diff
@@ -107,7 +113,7 @@ public void initializeDAO() throws Exception {
                 "myMethod description",
                 OffsetDateTime.now(),
                 methodVersionUUID,
-                "myMethod source"),
+                "GitHub"),
```

Comment:
this was the cause of the null pointer exception (see comment above)

Reply:
from @salonishah11 : can we just use the methodSource enum instead of a raw string?